service/src/main/java/com/epam/esm/service/exception/ext/IllegalArgumentException.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm.service.exception.ext;
+
+import com.epam.esm.service.exception.ServiceRuntimeException;
+import org.springframework.http.HttpStatus;
+
+public class IllegalArgumentException extends ServiceRuntimeException {
```

Comment:
java.lang.IllegalArgumentException? it would create a problem with class name for developers

Reply:
Renamed to 'InvalidRequestException'